src/test/java/core/basesyntax/service/impl/OperationHandlerTest.java

```diff
@@ -0,0 +1,86 @@
+package core.basesyntax.service.impl;
+
+import static org.junit.jupiter.api.Assertions.assertEquals;
+import static org.junit.jupiter.api.Assertions.assertThrows;
+
+import core.basesyntax.db.Storage;
+import core.basesyntax.model.FruitTransaction;
+import core.basesyntax.strategy.BalanceOperationHandler;
+import core.basesyntax.strategy.OperationHandler;
+import core.basesyntax.strategy.PurchaseOperationHandler;
+import core.basesyntax.strategy.ReturnOperationHandler;
+import core.basesyntax.strategy.SupplyOperationHandler;
+import org.junit.jupiter.api.AfterEach;
+import org.junit.jupiter.api.Test;
+
+public class OperationHandlerTest {

```

Comment:
Create a test class for each real class. The interface does not need test class.
![image](https://github.com/mate-academy/jv-fruit-shop-tests/assets/20326858/7a6785df-dc67-421b-9cd2-0716fdd6f297)